pages/index.js

```diff
@@ -1,7 +1,33 @@
+import useSWR from "swr";
+import Card from "@/components/Card";
+import styled from "styled-components";
+
 export default function HomePage() {
+  const { data } = useSWR("/api/plants");
+
+  if (!data) {
+    return <p>Loading...</p>;
+  }
+
   return (
     <div>
-      <h1>Hello from Next.js</h1>
+      <StyledPlantsList>
+        {data.map((plant) => {
+          return (
+            <li key={plant._id}>
+              <Card plant={plant} />
+            </li>
+          );
+        })}
+      </StyledPlantsList>
     </div>
   );
 }
+
+const StyledPlantsList = styled.ul`
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 1rem;
+  list-style: none;
+  padding: 0;
+`;
```

Comment:
please decide in a team whether your styled components go above or below the main component in the file and stick to one version.